src/main/scala/ai/privado/exporter/SinkExporter.scala

```diff
@@ -136,7 +136,7 @@ class SinkExporter(cpg: Cpg, ruleCache: RuleCache) {
                 .value
                 .dedup
                 .toArray
-              if (apiurls != null && apiurls.isEmpty) {
+              if (apiurls != null && (apiurls.isEmpty || apiurls.head == Constants.API)) {
```

Comment:
Apiurls is empty, so head can result in error?

Reply:
Updated to failsafe case